### ISSUE TITLE
Exclude *.pyc and *.pyo files from packaging

### ISSUE THREAD

**Issue by cwr10010** [ImmobilienScout24/pybuilder_aws_lambda_plugin#5]
I think we should not deliver *.pyc and *.pyo files within the zip by default to prevent incompatibility issues with the version of the runtime python in regards to the packaging python. If desired (eg for performance concerns) we can add a switch do allow delivery.

**Comment by mriehl** [comment]
AFAIK the CPython interpreter already deals with this problem itself (eG it regenerates the `py[c|o]` files if they are not suitable). Not sure about others (pypy, stackless) and also unsure if lambda persists the generated files.

**Comment by cwr10010** [comment]
Yes thats why we had a research in mind. as far as we know, lambda is a simple docker container, executing the python scripts and gets killed after a certain time... 

It is just about clean delivery. imho..
